packages/scene-composer/src/utils/entityModelUtils/sceneComponent.ts

```diff
@@ -170,3 +186,96 @@ export const updateSceneEntityComponent = (
+        throw new Error('Invalid rule statement');
+      }
+
+      ruleMap[id].statements.push(parsedStatement);
+    });
+  });
+
+  const dataBindingConfig: IDataBindingConfig = { fieldMapping: {}, template: {} };
+  const dataBindingConfigProp = comp.properties[SceneComponentProperty.PropertiesDataBindingConfig]?.value?.mapValue;
+  const fieldMappingProp = dataBindingConfigProp?.[DataBindingConfigPropertyKey.FieldMapping].mapValue;
+  if (fieldMappingProp) {
+    Object.keys(fieldMappingProp).forEach((key) => {
+      dataBindingConfig.fieldMapping[key] = JSON.parse(fieldMappingProp[key].stringValue || '');
+    });
+  }
+  const templateProp = dataBindingConfigProp?.[DataBindingConfigPropertyKey.Template].mapValue;
+  if (templateProp) {
+    Object.keys(templateProp).forEach((key) => {
+      if (templateProp[key].stringValue) {
+        dataBindingConfig.template![key] = templateProp[key].stringValue!;
+      }
+    });
+  }
+
+  const componentSettings: IComponentSettingsMap = {};
+  const componentSettingsProp = comp.properties[SceneComponentProperty.PropertiesComponentSettings]?.value?.mapValue;
+  if (componentSettingsProp?.[KnownComponentType.Tag]) {
+    const tagSettingsProp = componentSettingsProp[KnownComponentType.Tag].mapValue;
+    const scaleValue = Number(tagSettingsProp?.['scale']?.stringValue);
+    const tagSettings: ITagSettings = {
+      scale: isFinite(scaleValue) ? scaleValue : DEFAULT_TAG_GLOBAL_SETTINGS.scale,
+      autoRescale: tagSettingsProp?.['autoRescale']?.stringValue === 'true',
+      enableOcclusion: tagSettingsProp?.['enableOcclusion']?.stringValue === 'true',
+    };
+    componentSettings[KnownComponentType.Tag] = tagSettings;
+  }
+  if (componentSettingsProp?.[KnownComponentType.DataOverlay]) {
+    const overlaySettingsProp = componentSettingsProp[KnownComponentType.DataOverlay].mapValue;
+    const overlaySettings: IOverlaySettings = {
+      overlayPanelVisible: overlaySettingsProp?.['overlayPanelVisible']?.stringValue === 'true',
+    };
+    componentSettings[KnownComponentType.DataOverlay] = overlaySettings;
+  }
+
+  const document: ISceneDocumentInternal = {
+    version,
+    specVersion,
+    unit: comp.properties[SceneComponentProperty.Unit]?.value?.stringValue || DEFAULT_DISTANCE_UNIT,
+    ruleMap,
+    nodeMap: {},
+    componentNodeMap: {},
+    rootNodeRefs: [],
+    properties: {
+      [KnownSceneProperty.SceneRootEntityId]: entity.entityId,
+      [KnownSceneProperty.EnvironmentPreset]:
+        comp.properties[SceneComponentProperty.PropertiesEnvironmentPreset]?.value?.stringValue,
+      [KnownSceneProperty.DataBindingConfig]: dataBindingConfig,
+      [KnownSceneProperty.ComponentSettings]: componentSettings,
+      [KnownSceneProperty.MatterportModelId]:
+        comp.properties[SceneComponentProperty.PropertiesMatterportModelId]?.value?.stringValue,
+      [KnownSceneProperty.LayerIds]: comp.properties[SceneComponentProperty.ConnectedToLayers]?.value?.listValue?.map(
```

Comment:
In this cr, we still expect LayerIds or Layer exist, correct?

Reply:
yes